libqtile/backend/x11/window.py

```diff
@@ -1118,20 +1114,15 @@ def change_layer(self, up=True, top_bottom=False, kwargs=None):
             above = False
 
         # If the sibling is the current window then we don't need to configure stacking
-        # info so just pass on any additional kwargs and exit
         if sibling == self.wid:
-            if kwargs:
-                self.window.configure(**kwargs)
             return
 
         # Window needs new stacking info. We tell the server to stack the window
         # above or below a given "sibling"
-        if kwargs is None:
-            kwargs = {}
-        kwargs["stackmode"] = (
-            xcffib.xproto.StackMode.Above if above else xcffib.xproto.StackMode.Below
-        )
-        kwargs["sibling"] = sibling
+        kwargs = {
+            "stackmode": xcffib.xproto.StackMode.Above if above else xcffib.xproto.StackMode.Below,
+            "sibling": sibling
+        }
         self.window.configure(**kwargs)
```

Comment:
Maybe with only two static kwargs we can just do it inline, save the dictionary constructor:
```suggestion
        self.window.configure(
            stackmode=xcffib.xproto.StackMode.Above if above else xcffib.xproto.StackMode.Below,
            sibling=sibling,
	)
```